devops/llvm-anybuild.yml

```diff
@@ -0,0 +1,361 @@
+parameters:
+  # whether to build on Linux
+  - name: BuildOnLinux
+    type: boolean
+    default: true
+  # whether to build on Windows
+  - name: BuildOnWindows
+    type: boolean
+    default: true
+  # whether to build on MacOS
+  - name: BuildOnMacOs
+    type: boolean
+    default: false
+  # whether to build Verona
+  - name: UploadPackage
+    type: boolean
+    default: false
+  # extra command-line args to pass to AnyBuild
+  - name: ExtraAnyBuildArgs
+    type: string
+    default: --Verbose --WhyCacheMiss --WhyCacheMissOptions CacheDataStoreKey=LLVMVerona
+  # number of parallel cmake jobs when building LLVM on Linux
+  - name: CmakeJobsLinux
+    type: number
+    default: 48
+  # number of parallel cmake jobs when building LLVM on Windows
+  - name: CmakeJobsWindows
+    type: number
+    default: 24
+
+resources:
+  repositories:
+  - repository: self
+  - repository: llvm
+    type: github
+    endpoint: LLVMServiceConnection
+    name: llvm/llvm-project
+
+schedules:
+- cron: "0 3 * * *" # this is 3am UTC
+  displayName: Nightly AnyBuild LLVM Build
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+- name: AbEnvironmentName
+  value: LLVMVna
+- name: AbClusterId
+  value: 0310c9eb-18d0-4a58-bc85-bdc02a4d764a
+- name: AnyBuildEnvironmentUri
+  value: https://northeurope.anybuild-test.microsoft.com/api/clusters/$(AbClusterId)/agents
+- name: AnyBuildSource
+  value: https://anybuild$(AbEnvironmentName)neurope.blob.core.windows.net/clientreleases
+- name: AnyBuildRing
+  value: Dogfood
+- name: VeronaRepoGitUri
+  value: https://github.com/microsoft/verona
+- name: LlvmRepoGitUri
+  value: https://github.com/llvm/llvm-project
+# GOOD_HASH may be set in the pipeline in which case it overrides LLVM_COMMIT 
+# - name: GOOD_HASH
+#   value: ''
+
+jobs:
+### Configure: Determine commits to build
+###   - outputs VERONA_COMMIT and LLVM_COMMIT that correspond to the latest checked in change in verona@master
```

Comment:
`VERONA_COMMIT` is now unused. I'd remove all mentions of it.

Reply:
removed